packages/graphback-datasync/src/providers/DatasyncMongoDBDataProvider.ts

```diff
@@ -56,7 +56,7 @@ export class DataSyncMongoDBDataProvider<Type = any> extends MongoDBDataProvider
 
     data._deleted = true;
     const objectId = new ObjectId(idField.value);
-    const result = await this.db.collection(this.collectionName).updateOne({ _id: objectId, _deleted: false }, { $set: data });
+    const result = await this.db.collection(this.collectionName).updateOne({ _id: objectId, _deleted: { $ne: true } }, { $set: data });
```

Comment:
I do not understand reasoning behind this change. 
first - why we need to have _deleted here - It seems redundant. Also ne: true looks strange

Reply:
Hmm, I agree _deleted is redundant here